### DISCUSSION TITLE
Select specific set of fixtures conditionally

### DISCUSSION THREAD

**Discussion by f00dog-x300** [pytest-dev/pytest#10367] [Q&A]
Hi all, I hope the title makes sense. Essentially I was looking for a way to conditionally use a set of fixtures from a specific directory. 
This is essentially how the project is laid out: 

src
|
|_ ios
|
|_android
|
|_conftest.py

On my conftest.py file in the root of the project, I have something that looks like: 

pytest_plugins = [
    "pages.ios.pages.login_page_fixtures",
    "pages.ios.pages.status_page_fixtures",
]

I have similarly named test fixtures on both ios and android packages directories, however if I run an iOS test , I want to make sure that I only use the iOS test fixtures. However, later on I will need to use the android fixtures as part of the project. Since the fixtures are similarly named, I was wondering if there was a way to conditionally select the fixtures from ios vs android.

**Comment by RonnyPfannschmidt** [comment]
Currently there is no built-in mechanism, the recommended way to dynamically add plugins is to use the pytest_configure hook and registering plugins to the pluginmanager

**Reply by f00dog-x300** [reply]
Thank you for the answer. Is there an example I can take a look at?
